ogr/ogrsf_frmts/geojson/ogrgeojsonreader.cpp

```diff
@@ -3157,3 +3189,171 @@ json_object *CPL_json_object_object_get(struct json_object *obj,
     json_object_object_get_ex(obj, key, &poRet);
     return poRet;
 }
+
+bool OGRGeoJSONGetExtent(json_object *poObj, OGREnvelope *poEnvelope)
+{
+    if (!poEnvelope || !poObj)
+    {
+        return false;
+    }
+
+    // Get the "coordinates" array from the JSON object
+    json_object *poObjCoords = OGRGeoJSONFindMemberByName(poObj, "coordinates");
+
+    // Return if not found or not an array
+    if (!poObjCoords || json_object_get_type(poObjCoords) != json_type_array)
+    {
+        return false;
+    }
+
+    // poObjCoords can be an array of arrays, this lambda function will
+    // recursively parse the array
+    std::function<bool(json_object *, OGREnvelope *)> fParseCoords;
```

Comment:
``#include <functional>`` missing.
(or potentially transform this lamba as a static function outside of this function scope)